.gitattributes

```diff
@@ -54,5 +54,5 @@ tools/CoqMakefile.in whitespace=blank-at-eol
 # CR is desired for these Windows files.
 *.bat whitespace=cr-at-eol,blank-at-eol,tab-in-indent
 
-* eol=lf
-*.bat eol=crlf
```

Comment:
This causes line conversion in master which is why you see mingw.bat as crlf.

Reply:
I see. And because the .gitattribute in master also has *.bat eol=crlf I still see it as CR-LF when I do a pull - nasty. But I can see here:

https://raw.githubusercontent.com/coq/coq/master/dev/build/windows/MakeCoq_MinGW.bat

that in master it is indeed LF. So from my point of view the change is fine and I will approve.